doc/user/content/sql/isolation-level.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Isolation Level"
+description: "The isolation level defines how Materialize isolates the execution of transactions."
+aliases:
+ - /sql/consistency 
+menu:
+  main:
+    parent: 'reference'
+    weight: 160
+---
+
+The SQL standard defines four levels of transaction isolation. In order of least strict to most strict they are Read
+Uncommitted, Read Committed, Repeatable Read, and Serializable. In Materialize, you can request any of these isolation
+levels, but they all behave the same as the Serializable isolation level. In addition to the four levels defined in the
+SQL Standard, Materialize also defines a Strict Serializable isolation level.
+
+Isolation level is a per session configurable variable. To set the current session’s isolation level you can
+execute `SET TRANSACTION_ISOLATION TO '<isolation-level>';`. The default isolation level is Strict Serializable.
+
+## Serializable
+
+The SQL standard defines the Serializable isolation level as preventing the following three phenomenons:
+
+- **P1 (”Dirty Read”):**
+  > SQL-transaction T1 modifies a row. SQL-transaction T2 then reads that row before T1 performs a
```

Comment:
These quotes are directly from the SQL Standard, is there a good way to indicate that?

Reply:
Table or steps.

Table example:

```suggestion
    |   Time      |  T1        |  T2      |
    |-------------|------------|----------|
    |    1        |  Modify(x) |          |
    |    2        |            | Read(x)  |
    |    3        |  ROLLBACK  |          |
```

Reply:
Oh. Do you mean like a citation?

Reply:
Yeah I meant like a citation. Like what's the proper way to include a citation in our docs.

Reply:
A link to the particular section where the original citation appears is the best. If there is no public or easily accessible version, a link to the most reliable SQL Standard web or well-known access for the book should be enough. 

Examples:
[SQL Standard](https://www.iso.org/obp/ui/#iso:std:iso-iec:10026:-1:ed-2:v1:en)

Reply:
Just updated these with citations, let me know if it looks right.